docs/howto.md

```diff
@@ -1,18 +1,18 @@
-# How to
```

Comment:
In this case, we'll get the `Table of Contents` chapter in the left navigation menu.
So I guess it's better to leave the `How to` name here.

Here is a [pic](https://yadi.sk/i/Hz68b8NOlOqjxw) for a better understanding.